templates/eap72-amq-s2i.json

```diff
@@ -6,20 +6,20 @@
             "iconClass": "icon-eap",
             "tags": "eap,javaee,java,jboss,hidden",
             "version": "1.4.11",
```

Comment:
This version tag seems odd, the rest at 1.5.0. EAP CD 13 is currently using 1.5.0 for its template version, but I don't think that has an yimpact.

You can pick whatever you like to start these at, so 1.0.0 maybe? The xpaas label is usually set to the same value, as that is applied to the image in case we need to see what version of the template was used for the image.

Reply:
Thanks; sounds like I should change these to 1.0.0 as part of this PR. I believe these now are whatever was the value in CD 13 (not eap-cd-dev which might have moved since then.) But whatever they were the main point of this PR is to reset all the metadataish stuff from what was right for EAP CD to what's right for EAP on OS 7.2, so I should get this right.

Reply:
@luck3y so the corresponding xpaas label change would be here?

https://github.com/jboss-container-images/jboss-eap-7-openshift-image/pull/108/files#diff-bbc53605260cc5e485561cebece37165R20

Reply:
@bstansberry Yep, thats the one.